templates/pages/index.md.erb

```diff
@@ -36,8 +36,8 @@ I do my best, and I like it more than Russian, probably because of its laconism
 And I want to learn it more, and I think it's easier when living (or communicating)
 with English speakers.
 
-I use my devices (smartphone, PC, etc.) and all services (sites) in English,
-also I search information almost every time in English,
+I use my devices (smartphone, PC, etc.) and all services (sites) in English.
```

Comment:
I like the sentences separation. Why would I agree to a bigger complicated sentence?

Reply:
Yeah, why would you? (your initial sentence was bigger and more complicated than mine)

Reply:
OK, my bad. Probably, I should review this part (thought) one more time later.